File: app/src/main/java/com/autobridge/android/MainActivity.java
package com.autobridge.android;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.util.Log;

public class MainActivity extends Activity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        this.getApplicationContext().startService(new Intent(this.getApplicationContext(), Service.class));
    }
}

File: app/src/main/java/com/autobridge/android/MainActivity.kt
package com.autobridge.android

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.util.Log

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        this.applicationContext.startService(Intent(this.applicationContext, Service::class.java))
    }
}
